source/_docs/guides/jenkins.md

```diff
@@ -0,0 +1,312 @@
+---
+title: Use Jenkins to Test and Automate Drupal Development
+description: How to configure continuous integration for a Drupal site on Pantheon.
+tags: [siteintegrations]
+type: guide
+permalink: docs/guides/:basename/
+date: 6/20/2017
+---
+
+[Jenkins](https://jenkins.io) is an open source Continuous Integration (**CI**) server which can be used to build, test, and deploy code on any Drupal and WordPress website on Pantheon. Unlike hosted services such as [CircleCI](https://circleci.com/), it is a Java application installed and run on a server, and requires regular upkeep and maintenance. However, it is very customizable and can even run non-CI related tasks, such as calling periodic cron jobs.
+
+## What You Will Build
+
+This guide will cover how to configure an existing Jenkins server to work with sites on the Pantheon platform. Because we want to submit pull requests and take advantage of existing GitHub and Jenkins integration, we will configure GitHub as the main code repository. When we push code to the GitHub repo, it will trigger Jenkins to test our code changes on a multidev environment. Jenkins will run tests against both the pull request and Pantheon's master branch and display the results.
+
+## Before You Begin
+
+You will need:
+
+- Root access to a server running Jenkins.
```

Comment:
This is a big pre-req, with a high barrier to entry. Folks looking to get started with continuous integration will likely get stuck here before they even begin the guide.  Is there some way we can mitigate this?